cypress/integration/configuration/shippingMethods/createShippingMethod.js

```diff
@@ -19,15 +21,21 @@ import {
 } from "../../../support/pages/shippingMethodPage";
 
 filterTests({ definedTags: ["all"] }, () => {
-  describe("Create shipping method", () => {
+  describe("As a staff user I want to create shipping zone and rate", () => {
```

Comment:
This test fails during execution, probably changes to UI were made which resulted in error
![image](https://user-images.githubusercontent.com/97674352/153205412-0eaa4de6-3ad9-4a9f-a721-70bbc2bda366.png)